src/load_data.ado

```diff
@@ -58,52 +56,53 @@ This help file was dynamically produced by
 
 program define load_data 
 
-	syntax namelist(name=dataset), Years(numlist sort) [Vars(namelist)] [saveas(string)] [clear]
-	
-	if "${spdatapath}" == "" {
-		display as error "Global 'spdatapath' not found."
-		exit
-	}
+	syntax name(name=dataset), Years(numlist sort) [Vars(namelist)] [saveas(string)] [clear]
 	
-	if "`clear'" == "" & _N != 0 & c(changed) != 0 {
+    
+    * error checking ----------------------------------------------------------
+    
+    if "`clear'" == "" & _N != 0 & c(changed) != 0 {
 		display as error "no; dataset in memory has changed since last saved"
 		exit 4
 	}
-
-	local dataset = upper("`dataset'")
-	
-	if !inlist("`dataset'", "CPS", "ACS"){
-		display as error "dataset(`dataset') invalid. Must be CPS or ACS."
-	}
-
-	local vars = cond("`vars'" == "", "*", "`vars'")
-	
-	// confirm dataset is synched
-	mata : st_numscalar("dir_exists", direxists("${spdatapath}`dataset'"))
-	if scalar(dir_exists) != 1 {
-		display as error "${spdatapath}`dataset' not found. Make sure it is synched and try again"
-		exit 
-	}
-	
-	// confirm requested years are available for dataset
+    
+    // check supported dataset
+    local dataset = upper("`dataset'")
+    if !inlist("`dataset'", "CPS", "ACS"){
+		display as error "{bf:dataset()} must be acs or cps (case insensitive)"
+        exit 198
+	}    
+    // check years-dataset combination
 	if "`dataset'" == "CPS" {
 		capture numlist "`years'", range(>= 1980 <=2019)
 		if _rc != 0 {
-			display as error "years(`years') invalid. Datasets library has CPS data for 1980-2019."
-			exit
+			display as error "{bf:years()} must be between 1980 and 2019 inclusive when {bf:dataset()} is cps"
+			exit 198
 		}
 	}
 	if "`dataset'" == "ACS" {
 		capture numlist "`years'", range(>= 2000 <=2018)
 		if _rc != 0 {
-			display as error "years(`years') invalid. Datasets library has ACS data for 2000-2018."
-			exit
+			display as error "{bf:years()} must be between 200 and 2018 inclusive when {bf:dataset()} is acs"
```

Comment:
Should be `2000` instead of `200` (right?)

Reply:
Yes, thank you. I will plan to make another commit to this branch to fix this and the other issues you are finding.